docs/examples/basic_conceptual_guide/Part_2-dynamic_batching/Readme.md

```diff
@@ -0,0 +1,240 @@
+
+## Demonstration
+
+This section showcases the use of dynamic batching and concurrent model execution using the example for Part 1 of this series.
+
+### Getting access to the model
+
+Let's use the `text recognition` used in part 1. We do need to make some minor changes in the model, namely making the 0th axes of dynamic shape to enable batching. Step 1, download the Text Recognition model weights. Use the NGC PyTorch container as the environment for the following.
+```
+docker run -it --gpus all -v ${PWD}:/scratch nvcr.io/nvidia/pytorch:<yy.mm>-py3
+cd /scratch
+wget https://www.dropbox.com/sh/j3xmli4di1zuv3s/AABzCC1KGbIRe2wRwa3diWKwa/None-ResNet-None-CTC.pth
+```
+Export the models as .onnx using the file in the utils folder. This file is adapted from [Baek et. al. 2019](https://github.com/clovaai/deep-text-recognition-benchmark).
+```
+import torch
+from utils.model import STRModel
+
+# Create PyTorch Model Object
+model = STRModel(input_channels=1, output_channels=512, num_classes=37)
+
+# Load model weights from external file
+state = torch.load("None-ResNet-None-CTC.pth")
+state = {key.replace("module.", ""): value for key, value in state.items()}
+model.load_state_dict(state)
+
+# Create ONNX file by tracing model
+trace_input = torch.randn(1, 1, 32, 100)
+torch.onnx.export(model, trace_input, "str.onnx", verbose=True, dynamic_axes={'input.1':[0],'308':[0]})
+```
+
+### Launching the server
+
+As discussed in `Part 1`, a model repository is a filesystem based repository of models and configuration schema used by the Triton Inference Server (refer to `Part 1` for a more detailed explanation for model repositories). For this example the model repository structure would need to be set up in the following manner:
+```
+model_repository
+|   
+|-- text_recognition
+    |
+    |-- config.pbtxt
+    |-- 1
+        |
+        |-- model.onnx
+```
+This repository is a subset from the previous example. The key difference in this setup is the use of `instance_group`(s) and `dynamic_batching` in the model configuration. The additions are as follows:
+
+```
+instance_group [
+    {
+      count: 2
+      kind: KIND_GPU
+    }
+]
+dynamic_batching { }
+```
+With `instance_group` users can primarily tweak two things. First, the number of instances on a that model deployed on each GPU. The above example will deploy `2` intances of the model `per GPU`. Secondly, the target GPUs for this group can be specified with `gpus: [ <device number>, ... <device number> ]`.
+
+Adding `dynamic_batching {}` will enable the use of dynamic batches. Users can also add `preferred_batch_size` and `max_queue_delay_microseconds` in the body of dynamic batching to manage more efficient batching per their use case. Explore the [model configuration](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#model-configuration) documentation for more information
+
+As the model repository setup, the Triton Inference Server can be launched.
```

Comment:
A couple of potential typos here. Period after "more information". And this line might need to be reworded to have its meaning be clearer. Perhaps "With the model repository set up, the Triton..."